### PR TITLE
Drag n drop implemented

### PR DESCRIPTION
**Pull request by matroskin062**
# Drag n drop implemented
- Added data-attribute `data-card-id` for each card
- Drag n drop implemented using [Sortable.js](https://github.com/SortableJS/Sortable)  
- Added `PUT` method to CardAPI `updateCardStatus(id, status)`

### CODE REVIEW

File: src/js/ColumnUI.js [hunk 1]
```diff
@@ -30,6 +34,18 @@ class ColumnUI {
...
   }
+
+  dnd(cardList) {
+    const getStatus = () => this.status;
+    return new Sortable(cardList, {
+      group: 'shared',
+      animation: 150,
+      onAdd({ item }) {
+        const id = item.dataset.cardId;
+        CardAPI.updateCardStatus(id, getStatus());
```
Comment (alexgul1) [inline src/js/ColumnUI.js]:
```suggestion
        CardAPI.updateCard({
          id,
          status: getStatus()
        });
```

maybe better to use existing func `updateCard`

Comment (matroskin062) [reply src/js/ColumnUI.js]: > maybe better to use existing func `updateCard`

yep, sure

Comment (matroskin062) [reply src/js/ColumnUI.js]: fixed in f55bcf9

**Review by alexgul1 — APPROVED** [review]
good job

**Pull request merged**